opat/comparison.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>OPAT Comparison Tool</title>
+  </head>
+  <main>
```

Comment:
@mgifford looks like the main didn't work. Do you want to switch to iframes instead https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frameset since frameset is not recommended?